bokcirkel.py

```diff
@@ -32,7 +32,7 @@ def main():
         bot = Bot(intents=intents)
         bot.run(token)
 
-    except Exception as e:
+    except Exception:
         logging.exception("Bot error")
```

Comment:
not very informative

Reply:
The exception is added to the trace automatically with logging.exception.